lib/tasks/set_taxonomy_topic_email_override.rake

```diff
@@ -0,0 +1,25 @@
+desc "override email notifications for a document collection to a given taxonomy topic, can only be run on against a document that has never been published."
+task :set_taxonomy_topic_email_override, %i[document_collection_id taxon_content_id confirmation_string] => :environment do |_, args|
+  document_collection_id = args[:document_collection_id]
+  taxon_content_id = args[:taxon_content_id]
+  confirmation_string = args[:confirmation_string]
+
+  if document_collection_id.blank? || taxon_content_id.blank?
+    raise "Document collection ID and taxon content ID are required arguments"
+  end
+
+  document_collection = DocumentCollection.find_by(id: document_collection_id)
+  raise "Cannot find document collection with ID: #{document_collection_id}" unless document_collection
+
+  raise "Cannot set a taxonomy topic email override on previously published documents" if document_collection.document.live_edition_id.present?
```

Comment:
There's a [shorthand for this](https://github.com/alphagov/whitehall/blob/39df09dd26544ef5989dd162e1fa1bc7c123e72f/app/models/document.rb#L97) if you want it